src/test/ui/error-codes/E0621-does-not-trigger-for-closures.nll.stderr

```diff
@@ -11,7 +11,7 @@ LL |     invoke(&x, |a, b| if a > b { a } else { b }); //~ ERROR E0495
    |                ----------^^^^^-----------------
    |                |   |     |
    |                |   |     free region requires that `'1` must outlive `'2`
-   |                |   has type `&'1 i32`
+   |                |   lifetime `'1` appears in this argument
```

Comment:
so wait why this diff?

Reply:
oh because we don't find the `'1` at all?

Reply:
Updated most recent commit to address this.